drivers/counter/rtc_mcp7940n.c

```diff
@@ -757,6 +762,7 @@ static DEVICE_API(counter, mcp7940n_api) = {
 		},									\
 		.i2c = I2C_DT_SPEC_INST_GET(index),					\
 		.int_gpios = GPIO_DT_SPEC_INST_GET_OR(index, int_gpios, {0}),		\
+		.vbat_enable = DT_INST_PROP_OR(index, vbat_enable, false)		\
```

Comment:
For booleans you can just use `DT_INST_PROP()`, it will always be defined to true or false

Reply:
I didn't know about it, thanks! Fixed